corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -360,6 +360,16 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
             'keypress @ui.continueButton': 'continueAction',
         },
 
+        onRender: function () {
+            if (sessionStorage.selectedValues && sessionStorage.selectedValues.length !== 0) {
+                this.selectedCaseIds = sessionStorage.selectedValues.split(',');
+                this.updateCheckboxes();
+                this.selectedCaseIdsLength = this.selectedCaseIds.length;
```

Comment:
I don't know that this variable is pulling its weight. Noticed that [here](https://github.com/dimagi/commcare-hq/blob/95bd15477fcd1e54b23d324346ea36d8874799e0/corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js#L493) you're accessing the length directly which seems fine and should also work in the HTML template.

Reply:
adios 👋 [5b07b09](https://github.com/dimagi/commcare-hq/pull/31478/commits/5b07b09add01e66280eda31be5e1a798038882d1)